workflows/ampliseq.nf

```diff
@@ -97,6 +97,7 @@ include { MERGE_STATS                   } from '../modules/local/merge_stats'
 include { DADA2_TAXONOMY                } from '../modules/local/dada2_taxonomy'
 include { DADA2_ADDSPECIES              } from '../modules/local/dada2_addspecies'
 include { FORMAT_TAXRESULTS             } from '../modules/local/format_taxresults'
+include { FORMAT_TAXRESULTS as FORMAT_TAXRESULTS_ADDSP } from '../modules/local/format_taxresults'
```

Comment:
`FORMAT_TAXRESULTS_ADDSP` should inherit the setting in 
https://github.com/nf-core/ampliseq/blob/f1abc2010e849903f429a60e130da92b4ad2f8f8/conf/modules.config#L226-L232
from `FORMAT_TAXRESULTS` and the files should end up in the `results/dada2` folder. Just want to make sure that this is indeed the case (new syntax...), did you check?

Reply:
Yes, I checked and the files end up in the results/dada2 folder as before.

Reply:
Great, thanks, always good to learn whether it works as expected!